install/configuring_inventory_file.adoc

```diff
@@ -1734,17 +1734,30 @@ inventory file:
 ----
 openshift_enable_service_catalog=false
 ----
+
+If you disable the service catalog, the OpenShift Ansible broker and template
+service broker are not installed.
 ====
 
 When the service catalog is enabled, the OpenShift Ansible broker and template service broker are both enabled
 as well; see xref:configuring-openshift-ansible-broker[Configuring the OpenShift Ansible Broker] and xref:configuring-template-service-broker[Configuring the Template Service Broker] for more information.
```

Comment:
In the least, I think what we're adding can be combined with the sentence above the Ansible variable. So:

> [NOTE]
> ====
> To not install the OpenShift Ansible broker and the template service broker, set the service catalog variable in the Ansible inventory file at installation time to `false`:
> 
> ----
> openshift_enable_service_catalog=false
> ----
> ====

But, this section confuses me. It's called "Configuring the service catalog" and the only point is that it's enabled by default, and then there's a thing on how to disable it, but it's in a note box? If anything, I think "Enabling or Disabling the service catalog at installation" might be a better title, and getting rid of the note box.

Sorry, I realise this was like this when you got here...

Reply:
I think similar comments can be made about the other two sections you've touched below. Up to you how deep you want to go, though.